proxy/integration_test.go

```diff
@@ -41,10 +42,21 @@ func TestMain(m *testing.M) {
 	handlers["/v1/oneanddone"] = &oneAndDoneHandler{}
 	handlers["/v1/repeat"] = &repeatingHandler{}
 	handlers["/v1/sendafterclose"] = &sendAfterCloseHandler{}
+	handlers["/v1/hostStats/project"] = &statsHandler{1}
 	signedToken := test_utils.CreateBackendToken("1", privateKey)
 	url := "ws://localhost:1111/v1/connectbackend?token=" + signedToken
 	go backend.ConnectToProxy(url, handlers)
 
+	signedToken = test_utils.CreateBackendToken("2", privateKey)
+	handlers2 := make(map[string]backend.Handler)
+	handlers2["/v1/echo"] = &echoHandler{}
+	handlers2["/v1/oneanddone"] = &oneAndDoneHandler{}
+	handlers2["/v1/repeat"] = &repeatingHandler{}
+	handlers2["/v1/sendafterclose"] = &sendAfterCloseHandler{}
+	handlers2["/v1/hostStats/project"] = &statsHandler{2}
+	url = "ws://localhost:1111/v1/connectbackend?token=" + signedToken
+	go backend.ConnectToProxy(url, handlers2)
```

Comment:
I haven't thought about this too hard, so may be a dumb question but: why does the second backend need all these handlers? You only use /projects. Is it just for consistency?